oai_agents/common/tags.py

```diff
@@ -60,10 +62,18 @@ def map_to_index(teamtype):
             TeamType.SELF_PLAY: 9,
             TeamType.SELF_PLAY_LOW: 10,
             TeamType.SELF_PLAY_MEDIUM: 11,
-            TeamType.SELF_PLAY_HIGH: 12
+            TeamType.SELF_PLAY_HIGH: 12,
+            TeamType.SELF_PLAY_ADVERSARY: 13
         }
         return tt_map[teamtype]
 
 class TeammatesCollection:
     TRAIN = 'train'
     EVAL = 'eval'
+
+class CheckedPoints(Enum):
```

Comment:
I also think this should be called EvalCheckedPoints to be more readable.
Also, please add a comment that explains what this is used for